assets/js/react/hooks/react-component.tsx

```diff
@@ -148,10 +151,10 @@ export const ReactComponent = {
   },
 
   _onBoundary(element) {
-    this.__view.liveSocket.requestDOMUpdate(() => {
+    this.__view().liveSocket.requestDOMUpdate(() => {
       if (element == null) return;
 
-      this.__view.execNewMounted();
+      this.__view().execNewMounted();
```

Comment:
Ah - presumably this was the change that you were stuck on before?

Reply:
Yeah, for some unknown reason. The internal implementation of a phoenix live's view got changed from an object to a function that then returns the object. Seems we were on a previous version during Xandor's implementation & after the version update it started erroring. 
@stuartc sent a git diff which fixed it